creator-node/src/models/file.js

```diff
@@ -27,6 +27,16 @@ module.exports = (sequelize, DataTypes) => {
       type: DataTypes.TEXT,
       allowNull: true
     },
+    // the queryable filename if this file entry is inside a dir
```

Comment:
maybe clarify in comment this is only for files of type "image" else null

Reply:
Added a comment